src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -288,40 +238,22 @@ describe('<ComboBox/>', () => {
 
     it('renders a custom given input', () => {
       const comboBox = mountWithAppProvider(
-        <ComboBox
-          options={options}
-          selected={[]}
-          textField={<input type="text" />}
-          onSelect={noop}
-        />,
+        <ComboBox {...mockProps} textField={<input type="text" />} />,
       );
       expect(comboBox.find('input')).toHaveLength(1);
       expect(comboBox.find(TextField)).toHaveLength(0);
```

Comment:
This test will also pass if you render a `TextField` so you might wanna test this differently e.g.

```ts
const input = <input type="text" />;
const comboBox = mountWithAppProvider(<ComboBox {...mockProps} textField={input} />);
expect(comboBox.contains(input)).toBeTruthy()
```